libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -154,6 +154,13 @@ void IRGeneratorForStatements::initializeStateVar(VariableDeclaration const& _va
 	}
 }
 
+void IRGeneratorForStatements::initializeLocalVar(VariableDeclaration const& _varDecl)
+{
+	solAssert(m_context.isLocalVariable(_varDecl), "Must be a local variable.");
+	IRVariable zero = zeroValue(*_varDecl.type());
```

Comment:
Please check that this works for storage pointers :)

Reply:
Don't storage pointers need to always be initialized?

Reply:
Only before they are used.